engine/common/system.c

```diff
@@ -91,7 +91,7 @@ write screenshot into clipboard
 */
 void Sys_SetClipboardData( const char *buffer, size_t size )
 {
-	Platform_SetClipboardText( buffer, size );
+	Platform_SetClipboardText( buffer );
```

Comment:
Sys_SetClipboardData function is not used anywhere. Let's remove it.

Reply:
Ok